### DISCUSSION TITLE
Help wanted about Python 3

### DISCUSSION THREAD

**Discussion by Saikatsaha1996** [termux/termux-packages#10883] [Q&A] [answered]
I have installed `python-3.10.4`

But i want to downgrade to `python3.6`

Termux 0.118
aarch64

How can I do ?
Help appreciate

Thank you
Saikat

**Comment by ghost** [comment]
Any specific reasons why you want to downgrade a python version?

**Reply by Saikatsaha1996** [reply]
[silentarmy.txt](https://github.com/termux/termux-packages/files/8832710/silentarmy.txt)
> Any specific reasons why you want to downgrade a python version?

I compiled some software.. but not running with python 3.10
/data/data/com.termux/files/home/AMDVerusCoin/./silentarmy.py:191: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead                                                                 def reconnect(self):                                                      /data/data/com.termux/files/home/AMDVerusCoin/./silentarmy.py:210: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead                                                                 def show_stats(self):                                                     /data/data/com.termux/files/home/AMDVerusCoin/./silentarmy.py:308: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead                                                                 def start_solvers(self, devid):                                           /data/data/com.termux/files/home/AMDVerusCoin/./silentarmy.py:184: DeprecationWarning: There is no current event loop                                     self.loop = asyncio.get_event_loop()                                      /data/data/com.termux/files/home/AMDVerusCoin/./silentarmy.py:46: DeprecationWarning: There is no current event loop                                      loop = asyncio.get_event_loop()


I don't know any programing language..
And mey i need to edit some code.. without help i can't do anything.. thats why i want..

@wmcbtech30 thank you for your response.. unofficial reason but want solution ..
Waiting for solution

**Comment by 2096779623** [comment, marked as answer]
https://github.com/Termux-pod/termux-pod

There are previous versions of Python here, but without Python 3.6, try compiling with this Build.sh: https://github.com/termux/termux-packages/blob/d33909e9d18a6f20b15950cb31d68241fba91fce/packages/python/build.sh

**Reply by Saikatsaha1996** [reply]
Thank you for your help
